uber/site_sections/preregistration.py

```diff
@@ -347,7 +347,9 @@ def unset_group_member(self, session, id):
 
     def add_group_members(self, session, id, count):
         group = session.group(id)
-        if int(count) < group.min_badges_addable:
```

Comment:
Since we're leaving Uber fully active and accessible, we don't want people to be able to add badges to their group to get the group price on new registrations.  (Or maybe we do?  I'm assuming that we do not, my opinion is that we shouldn't allow this.)